test/cppsuite/test_harness/workload/thread_context.cxx

```diff
@@ -80,21 +81,22 @@ transaction_context::try_begin(const std::string &config)
         begin(config);
 }
 
-void
+/* It's possible to receive rollback in commit which is handled internally. */
+bool
 transaction_context::commit(const std::string &config)
 {
+    WT_DECL_RET;
```

Comment:
Can we try to have only one `return` statement in a function ? We should return `_needs_rollback` at the end of this one.

Reply:
ed87160